### PR TITLE
Improve market

### PR DESCRIPTION
**Pull request by samogot**
- If there is no sell orders for some resource, try to make one with high price
  It is very useful for shard1 where market is weak
- Add balance coefficient to choose between distant room with high price and close room with low price, when selling to other's by orders

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -204,8 +204,16 @@ global.config = {
...
+    minAmount: 5000
+  },
+
+  market: {
+    minAmount: 100000,
+    energyCreditEquivalent: 1,
+    trySellOrders: true,
+    sellOrderMaxAmount: 100,
+    sellOrderReserve: 2000,
+    sellOrderPriceMult: 5
```
Comment (TooAngel) [inline src/config.js]: I guess `Mult` means `Multiplicator`, mind writing the full word, doesn't cost anything and easier to grasp the meaning

**Comment by democratic-collaboration** [comment]
A new review, yeah.

    Votes: 271/456
    Coefficient: 0.594298245614
    Merging in 4 days 1 hours
    Age 0 days 0 hours

**Comment by TooAngel** [comment]
Well fixing the codeclimate issues would be nice

**Comment by samogot** [comment]
refactored)

**Comment by democratic-collaboration** [comment]
A new review, yeah.

    Votes: 296/459
    Coefficient: 0.644880174292
    Merging in 2 days 12 hours
    Age 1 days 0 hours

**Comment by democratic-collaboration** [comment]
A new review, yeah.

    Votes: 300/463
    Coefficient: 0.647948164147
    Merging in 0 days 6 hours
    Age 3 days 6 hours

**Pull request merged**